src/pluginutils.c

```diff
@@ -52,6 +52,16 @@ typedef struct
 PluginDocDataProxy;
 
 
+/** Returns the runtime API version Geany was compiled with.
+ *
+ * @return Geany's API version
```

Comment:
Maybe add more detailed docs, like "this can be used to [blah blah what is it useful for] at runtime"?  @elextr !!!